src/Components/CoffeeStoreContainer.tsx

```diff
@@ -1,27 +1,41 @@
 import { spinnerService } from '@simply007org/react-spinners';
 import React, { useEffect, useState } from 'react';
-import { translate } from 'react-translate';
 import { Client } from '../Client';
 import { matchesTaxonomy } from '../Utilities/CheckboxFilter';
-import { defaultLanguage, initLanguageCodeObject } from '../Utilities/LanguageCodes';
+import { defaultLanguage, initLanguageCodeObjectWithArray } from '../Utilities/LanguageCodes';
 import CheckboxFilter from './CheckboxFilter';
 import CoffeeStoreListing from './CoffeeStoreListing';
+import { ITaxonomyTerms } from '@kentico/kontent-delivery';
+import { Coffee } from '../Models/coffee';
+import { useIntl } from 'react-intl';
+import { projectModel } from '../Models/_project';
 
-const CoffeeStoreContainer = ({ language, t }) => {
-  const [coffees, setCoffees] = useState(initLanguageCodeObject());
-  const [processings, setProcessings] = useState([]);
-  const [productStatuses, setProductStatuses] = useState([]);
+interface CoffeeStoreContainerProps {
```

Comment:
Can we use React Intl to load language?

Reply:
[51a7380](https://github.com/Kentico/kontent-sample-app-react/pull/206/commits/51a738021b8ce01bfe50c4dfb9a1592e744ca2d2)